provider-tutorials/provider-cli.md

```diff
@@ -214,13 +214,150 @@ For `--network`you have two options, either `mainnet` or `rinkeby`. For `--to-ad
+
+This feature defines rules and modes for Outbound Network.
+It gives provider control over which entities are trusted and what level of Outbound Network access they have.
+
+Outbound feature provides three rules:
+
+1. `everyone`: defines the default outbound access for all requestors.
+2. `audited-payload`: controls outbound access for manifests which are audited and signed by a trusted party.
+   Trust is gained when provider sets this rule for root certificate of auditing organisation.
+   Provider can have multiple `audited-payload` rules for different `X.509` certificates.
+   To match this rule, requestor should get his manifest audited and signed by organisation which is trusted by provider.
+   It doesn't matter from which requestor node request is made.
+3. `partner`: allows outbound access for any manifests, as long as requestor node is a partner to trusted organisation.
+   Trust is gained when provider sets Partner rule for root certificate of trusted organisation.
+   Provider can have multiple `partner` rules for different [golem certificates](https://github.com/golemfactory/golem-architecture/blob/master/gaps/gap-25_golem_certificates/gap-25_golem_certificates.md).
+   To match this rule, requestor should get his [node descriptor](https://github.com/golemfactory/golem-architecture/blob/master/gaps/gap-31_node_descriptor/gap-31_node_descriptor.md) signed by partner to trusted organisation.
+   It doesn't matter which payload is being requested.
+
+Every rule can operate in three different modes:
+
+1. `all`: all domains listed in the manifest can be accessed for outbound traffic.
+2. `whitelist`: only whitelisted domains can be accessed for outbound traffic.
+3. `none`: no outbound traffic is allowed.
+
+> **Warning**
+> Remember, it is your responsibility as a provider to decide which entities you trust and the level of Outbound Network access you want to enable for them.
+
+By default, the following rules and modes are set during installation:
+
+- Everyone: Whitelist mode (every requestor can access outbound until domain is whitelisted).
+- AuditedPayload: All mode (Golem root certificate is trusted).
+- Partner: All mode (another Golem root certificate is trusted).
+
+#### Listing Rules
+
+To list currently set rules, you can use the `ya-provider rule list` command. By default, it displays the rules in a table format:
+
+```
+$ ya-provider rule list
+Outbound status: enabled
+┌──────────────────┬─────────────┬───────────────┬───────────────┐
+│  rule            │  mode       │  certificate  │  description  │
+├──────────────────┼─────────────┼───────────────┼───────────────┤
+│  Everyone        │  whitelist  │               │               │
+│  AuditedPayload  │  all        │  55e451bd     │               │
+│  Partner         │  all        │  80c84b27     │               │
+└──────────────────┴─────────────┴───────────────┴───────────────┘
+```
+
+You can also retrieve the rules in JSON format by adding the `--json` flag:
```

Comment:
Every `ya-provider` command printing table has `--json` flag, so maybe we can skip it?
A valid reason for keeping it is this issue with `certificate` id columns values, which can be different between `keystore list` and `rule list`, so it at least shows how to get full id.
Still it is unlikely anyone will face it.

Reply:
Yes, it can be shorter

Reply:
So I propose that it will stay here so that user can have easy way of finding big cert-id's so it can be used in keystore